analysis/results.csv

```diff
@@ -0,0 +1,20 @@
+1 - 4, 7, 21, 58, 37, 7, 21, 14, 20.58823529411765, 61.76470588235294, 41.1764705882353, 4.857142857142857, 1.619047619047619, 2.4285714285714284, 310, 800, 568, 911.7647058823529, 2352.9411764705883, 1670.5882352941176, 0.10967741935483871, 0.0425, 0.05985915492957746, 3, 7, 2, 3, 1, 1
```

Comment:
What's this file showing?

Reply:
oh i probably could remove this - its the csv data that populated our excel sheet.

Reply:
oh I see. Yea I would say remove this one.